doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -99,25 +101,16 @@ may have been accessed and changed by someone else and a push will fail::
     $ git branch <new version>  # only when starting new numpy version
     $ git checkout v1.14.x  # v1.14.x already existed for the 1.14.4 release
 
-Edit the ``.travis.yml`` and ``.appveyor.yml`` files to make sure they have the
-correct version, and put in the commit hash for the ``REL`` commit created
-above for ``BUILD_COMMIT``, see the _example from `v1.14.3`::
-
-    $ gvim .travis.yml .appveyor.yml
-    $ git commit -a
-    $ git push upstream HEAD
+Edit the ``azure/posix.yml`` and ``azure/windows.yml`` files to make sure they
+have the correct version, and put in the commit hash for the ``REL`` commit
+created above for ``BUILD_COMMIT``. Push and, if you are on a fork of the repo,
+merge the branch.
```

Comment:
I prefer the command version for cut and paste. In general, I just cut and paste for a release. It is also clearer to demonstrate than to explain. I'd just use the two new file names instead of the old ones.

Reply:
ok, also copied to `HOWTO_release.rst`